src/widget/text.rs

```diff
@@ -9,32 +13,32 @@ pub struct Options<Color> {
 #[derive(Clone, Copy)]
 pub struct Text<Color> {
     options: Options<Color>,
+    text: &'static str,
 }
 
 impl<Color> Text<Color>
 where
     Color: PixelColor,
 {
-    pub fn new(options: Options<Color>) -> Self {
-        Self { options }
+    pub fn new(options: Options<Color>, text: &'static str) -> Self {
+        Self { options, text }
     }
 }
 
-impl<Color> Widget<Color> for Text<Color>
+impl<Color, Display> Widget<Display> for Text<Color>
```

Comment:
I don't think I understand why `Display` is generic here - this isn't a "blanket" implementation, right? This is the implementation of the `Widget` trait for the `Text` struct? But so if you have an instance of `Text` is there just one type of `Display` that would get inferred if you call `.draw()` on it? Or is this generic saying "you can call `.draw()` and pass it any `display` whose type satisfies `Display: DrawTarget<Color = Color>`"?

Reply:
> But so if you have an instance of Text is there just one type of Display that would get inferred if you call .draw() on it?

Yes, which seems bad, but I couldn't figure out a way around it while keeping `Widget` object-safe. In practice that's probably fine since it's very unlikely you would need to use that same instance of a widget on two different displays?